fields.yml

```diff
@@ -381,35 +381,55 @@
             Unique ID to describe the event.
           example: 8a4f500d
     
+        - name: kind
+          level: core
+          type: keyword
+          description: >
+            The kind of the event.
+    
+            This gives information about what type of information the event
+            contains, without being specific to the contents of the event.  Examples
+            are `event`, `state`, `alarm`.
+          example: state
+    
         - name: category
           level: core
           type: keyword
           description: >
             Event category.
     
-            This can be a user defined category.
-          example: metrics
+            This contains high-level information about the contents of the event. It
+            is more generic than `event.action`, in the sense that typically a
+            category contains multiple actions.
+          example: user-management
     
-        - name: type
+        - name: action
           level: core
           type: keyword
           description: >
-            A type given to this kind of event which can be used for grouping.
+            The action captured by the event.
     
-            This is normally defined by the user.
-          example: nginx-stats-metrics
+            This describes the information in the event. It is more specific than
+            `event.category`. The value is normally defined by the user.
+          example: user-password-change
     
-        - name: action
+        - name: outcome
```

Comment:
`outcome` is also meant to be a field with prescribed content. We should add a similar warning as what I suggest for `event.category`. More or less "We're about to come up with a list of prescribed values, so use with caution."

Reply:
warning added.